src/Files/App.xaml.cs

```diff
@@ -315,17 +320,30 @@ private Frame EnsureWindowIsInitialized()
             return rootFrame;
         }
 
-        private void CoreWindow_Activated(CoreWindow sender, WindowActivatedEventArgs args)
+        private async void CoreWindow_Activated(CoreWindow sender, WindowActivatedEventArgs args)
         {
             if (args.WindowActivationState == CoreWindowActivationState.CodeActivated ||
                 args.WindowActivationState == CoreWindowActivationState.PointerActivated)
             {
                 ShowErrorNotification = true;
-                ApplicationData.Current.LocalSettings.Values["INSTANCE_ACTIVE"] = Process.GetCurrentProcess().Id;
+
                 if (MainViewModel != null)
                 {
                     MainViewModel.Clipboard_ContentChanged(null, null);
                 }
+
+                var proc = Process.GetCurrentProcess();
+                ApplicationData.Current.LocalSettings.Values["INSTANCE_ACTIVE"] = proc.Id;
```

Comment:
TODO: docs suggest using `VisibilityChanged` instead of `Activated` to check if the prelaunch instance has been opened.